lib/wallet/api/http/Cardano/Wallet/Api/Types.hs

```diff
@@ -883,7 +883,7 @@ newtype ApiWalletUtxoSnapshot = ApiWalletUtxoSnapshot
 
 data ApiWalletUtxoSnapshotEntry = ApiWalletUtxoSnapshotEntry
     { ada :: !(Quantity "lovelace" Natural)
-    , adaMinimum :: !(Quantity "lovelace" Natural)
+    , adaMinimum :: !(Maybe (Quantity "lovelace" Natural))
```

Comment:
I propose that we just remove this field entirely. (See https://github.com/cardano-foundation/cardano-wallet/pull/4084)